lib/diagnostic.js

```diff
@@ -41,7 +71,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.deleteMany({motto: "As High as Honor"});
```

Comment:
This will delete the entire document.  We just want to delete the key from the document.  This can be done by using `update()` with `$unset`.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```